pycbc/events/events.py

```diff
@@ -220,6 +220,21 @@ def newsnr_sgveto(snr, bchisq, sgchisq):
     else:
         return nsnr[0]
 
+def newsnr_sgveto_psdvar(snr, bchisq, sgchisq, psd_var_val):
+    """ Combined SNR derived from NewSNR, Sine-Gaussian Chisq and PSD 
+    variation statistic """
+    nsnr = newsnr_sgveto(snr, bchisq, sgchisq)
+    if psd_var_val >= 1.8:
```

Comment:
does this work for an array of snrs, psd_var_val's etc?  Might want to look at what is done in newsnr and newsnr_sgveto to 'numpy-proof' the functions.

Reply:
I took a look at some of the other functions and copied what they did. Should be consistent now - thanks for the catch